src/sys.zig

```diff
@@ -662,6 +662,15 @@ pub fn close(fd: bun.FileDescriptor) ?Syscall.Error {
     return bun.FDImpl.decode(fd).close();
 }
 
+pub fn close2(fd: bun.FileDescriptor) ?Syscall.Error {
```

Comment:
can this be renamed to `closeIgnoreStdio`, and then moved into FDImpl following the logic in `pub fn close` there. though i wonder if the original guard on this should be ignoring stdin too.

with my changes in one branch this snippet will not compile on windows